src/js/components/RegisterPage/SignUp.js

```diff
@@ -155,7 +155,7 @@ class SignUp extends Component {
 
                                 <div className="tab-content">
                                     <div className="tab-pane container active" id="home">Some interesting information, when I say interesting I mean really interesting</div>
-                                    <div className="tab-pane container" id="menu1">Some interesting information, when I say interesting I mean really interesting</div>
+                                    <div className="tab-pane container" id="menu1">Thought I found a way , Isn't it lovely say interesting I mean really interesting</div>
```

Comment:
What is with the repetition of menu buttons ? 
Same text being set over and over again. 
Add to const var and use the variable

Reply:
Thanks I resolved this by making them distinct